adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Constants.cs

```diff
@@ -167,6 +166,9 @@ internal static class AdalErrorMessage
         public const string DeviceCertificateNotFoundTemplate = "Device Certificate was not found for {0}";
 
         public const string InteractionRequired = "interaction_required";
+
+        public const string GetUserNameFailed = "Failed to get user name";
```

Comment:
>public const s [](start = 8, length = 14)

not sure if this change is needed (at least I do not understand it)

Reply:
git conflict resolution ....

---
In reply to: [221416051](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1284#discussion_r221416051) [](ancestors = 221416051)